### PR TITLE
Add ALP job group

### PR DESCRIPTION
**Pull request by jlausuch**
This is the first draft for running basic tests for ALP.
Related ticket: https://progress.opensuse.org/issues/112418

### CODE REVIEW

**Review by mloviska — APPROVED** [review]
LGTM

File: job_groups/alp_poc.yaml [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+defaults:
+  x86_64:
+    machine: 64bit
+    priority: 50
+  aarch64:
+    machine: aarch64
+    priority: 50
+    settings:
+      QEMUCPUS: '2'
```
Comment (ggardet) [inline job_groups/alp_poc.yaml:45]: Any reason to use 2 cpus on aarch64 only?

Comment (ggardet) [reply job_groups/alp_poc.yaml:45]: Yes, by default this is 1 cpu. But this is the same for x86. 
I would try to keep all archs settings as close as possible, so trying with the default `QEMUCPUS=1` looks better to me. But we can revisit if things are not as good as expected.

Comment (jlausuch) [reply job_groups/alp_poc.yaml]: that makes a lot of sense, thank you! :)

**Comment by lkocman** [comment]
will be turned into news-o-o article: https://etherpad.opensuse.org/p/lkocman-draft-qa-alp

**Pull request merged**